admin_manual/installation/system_requirements.rst

```diff
@@ -24,14 +24,13 @@ Supported Platforms
 
 * Server: Linux (Debian 7, SUSE Linux Enterprise Server 11 SP3 & 12, 
   Red Hat Enterprise Linux/Centos 6.5 and 7 (7 is 64-bit only), Ubuntu 12.04 
-  LTS, 14.04 LTS, 14.10)
-* Web server: Apache 2 with mod_php
+  LTS, 14.04 LTS, 14.10, 15.10, 16.04)
+* Web server: Apache 2.4 with mod_php
```

Comment:
Drop and add some old and new Linuxes. The entry should read like this to match the packages configured in the build service:

* Server: Linux (Debian 7 + 8, SUSE Linux Enterprise Server 12 & SP1, 
   Red Hat Enterprise Linux/Centos 6.5 and 7 (7 is 64-bit only), Ubuntu 14.04 LTS, 15.10, 16.04 LTS)